modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -575,7 +575,7 @@ SHOW ALIAS `remote-northwind-2021` FOR DATABASE YIELD name, properties
 
 
 Database aliases can be altered using `ALTER ALIAS` to change its database target, properties, URL, user credentials, or driver settings.
-The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Access control -> DBMS administration].
```